samples/subsys/mgmt/mcumgr/smp_svr/README.rst

```diff
@@ -47,6 +47,14 @@ Bluetooth Low Energy (BLE) and do not have a built-in or pluggable BLE radio,
 you can build one and use it following the instructions in
 :ref:`bluetooth-hci-uart-bluez`.
 
+Building and Running using NCS
+******************************
+Building the sammple as a part off nRF Connect SDK is simplified in comparasion
+to the vanilla zephyr sample.
+
+.. include:: /includes/build_and_run.txt
+
+
 Building and Running
```

Comment:
I would remove original zephyr build and Running section.
Or maybe It would be better to rename it to Building and Running (vanilla zephyr) or something which reflect that this is zephyr base build manual.
What are yours opinion?

Reply:
I think we should keep it, at the very least to avoid merge conflicts. @ru-fu ?

Reply:
No strong opinion here. If we want people to use the NCS way of building, we should remove this section. But if that leads to constant merge conflicts, I don't mind keeping it.